src/main/java/com/gizasystems/automationexercise/pages/ProductQuantityPage.java

```diff
@@ -26,7 +26,11 @@ public ProductQuantityPage clickOnViewProduct() {
         GoogleAlert.dismissAlert(driver, viewProductbtn_button);
         return this;
     }
-
+    @Step("Verify that Product details is opened")
+    public ProductQuantityPage verifyOnProductDetails() {
+        driver.element().assertThat(productDetails).exists().perform();
+        return this;
+    }
```

Comment:
Why did we move this validation method from bottom? As agreed, Validation methods should be at the bottom of the class as it was before.

Reply:
@MahmoudElSharkawyGS  done , please review again